src/show/components/TextField/index.js

```diff
@@ -37,7 +49,7 @@ TextField.propTypes = {
   placeholder: PropTypes.string,
   required: PropTypes.bool,
   type: PropTypes.string,
-  value: PropTypes.string
+  value: PropTypes.any
```

Comment:
How is this any? It can be one of number and string.

Reply:
capis